nextflow.config

```diff
@@ -115,14 +112,18 @@ params {
     // --------------
     libtype = 'A'
     // gene_feature_gff_to_create_transcriptome_host   = ["exon"] // old
-    gene_feature_gff_to_create_transcriptome_host   = "exon" // new
+    gene_feature_gff_to_create_transcriptome_host = "exon"
+    // new
     gene_attribute_gff_to_create_transcriptome_host = "transcript_id"
     // gene_feature_gff_to_create_transcriptome_pathogen = ["gene", "sRNA", "tRNA", "rRNA"] //old
```

Comment:
If yo are wondering why there are a bunch of these comments that I've commented out and left. I've kept them in until all three tools successfully run and the output looks reliable. Otherwise if something isn't quite correct somewhere, these are a quick reference point